src/crimson/osd/object_context.h

```diff
@@ -124,9 +130,12 @@ class ObjectContext : public ceph::common::intrusive_lru_base<
     });
   }
 
-  boost::intrusive::list_member_hook<> list_hook;
+  boost::intrusive::list_member_hook<> obc_accessing_hook;
   uint64_t list_link_cnt = 0;
+  bool fully_loaded = false;
+  bool invalidated_by_interval_change = false;
```

Comment:
I'm trying to work on increasing the parallelism to load missing obcs from a PG.

Will try to further clean up the obc state using named enum.

Reply:
They are actually orthogonal -- it's possible for invalidated_by_interval_change to be set prior to fully_loaded.  Probably simpler to keep them as enums since they are purely for debugging.